modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -107,35 +149,54 @@ image::workspaces/configure-command.png[Configure tasks]
 
 
 [id="importing-to-a-running-workspace-using-the-git-clone-command_{context}"]
-== Importing to a running workspace using the *Git: Clone* command
+== Importing a project with *Git: Clone* in the GUI
+
+You can import your project into a running workspace with the *Git: Clone* command in the graphic user interface of {prod-short}.
+ 
+.Prerequisites
+* A started workspace
 
-To import to a running workspace using the *Git: Clone* command:
+.Procedure
 
-. Start a workspace, then use the *Git: Clone* command from the command palette or the Welcome screen to import a project to a running workspace.
+Import your project's codebase into a running workspace as follows:
+
+. Select the *Git: Clone* command on the *Welcome* screen or from the command palette:
+* On the *Welcome* screen, click the *Git: Clone* link.
 +
 image::workspaces/{project-context}-welcome.png[Welcome screen]
 
-. Open the command palette by using `F1`, `CTRL-SHIFT-P`, or the link displayed on the Welcome screen.
+* To open the command palette, press F1 or Ctrl+Shift+P.
```

Comment:
or kbd:[Ctrl+Shift+p].

Reply:
Great, thank you, I didn't know about this! Changed.